examples/sasl_consumer.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright 2020 Confluent Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+#
+# This is a simple example of the DeserializingConsumer using SASL authentication.
+#
+import argparse
+
+from confluent_kafka import DeserializingConsumer
+from confluent_kafka.serialization import StringDeserializer
+
+
+def sasl_conf(args):
+    """
+    Generates a SASL configuration dict from a Namespace instance
+
+    Args:
+        args (argparse.Namespace): parsed args
+
+    Returns:
+         dict: SASL configuration dict
+
+    """
+    sasl_mechanism = args.sasl_mechanism
+
+    sasl_conf = {'sasl.mechanism': sasl_mechanism,
+                 'security.protocol': 'SASL_PLAINTEXT'}
```

Comment:
Might want to make a note here for SASL_SSL

Reply:
mention or add support for

Reply:
Mentioning when to use SASL_SSL  is enough.